spyder_terminal/terminalplugin.py

```diff
@@ -20,7 +20,7 @@
 from qtpy.QtCore import Qt, Signal, QTimer, Slot
 from qtpy.QtGui import QKeySequence
 
-from spyder.plugins import SpyderPluginWidget
+from spyder.api.plugins import SpyderPluginWidget
```

Comment:
Please make things to work with Spyder 3 and 4, like this

https://github.com/spyder-ide/spyder-notebook/blob/d4530aef67bec00cc98b294fc3694540636c7207/spyder_notebook/notebookplugin.py#L41-L46